### PR TITLE
Add Ai -> Machine Learning

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**

### CODE REVIEW

**Comment by enki-hq** [comment]
Failed parsing files: 
  - ai/machine-learning/neural-networks/neural-networks.md: 
 > Missing compiler for node of type `questionGap`: `[object Object]`

File: ai/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+name: Ai
+
+description: Ai's and Machine Learning
```
Comment (nem035) [inline ai/README.md:3]:
```suggestion
name: AI

description: Understand how intelligent systems work, and use them to solve real-world problems.
```

File: ai/machine-learning/README.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+name: Introduction to ML
+
+description: Explore various types of machine learning algorithms such as supervised learning, unsupervised learning, and more.
```
Comment (nem035) [inline ai/machine-learning/README.md:3]:
```suggestion
name: Intro to ML

description: Understand how machines can learn and make decisions just like humans!
```

**Comment by enki-hq** [comment]
### Course

| Course Name | Error Message |
|:-:|:-:|
| machine-learning | We couldn't find these workouts: `learning-by-doing`, `learning-by-chasing-a-reward`. |
### Topic

| Topic Name | Error Message |
|:-:|:-:|
| ai | Topic color `#0984E3` does not conform to our HSV rules. Replace it with `#a6d8ff`. |
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |
### Insight

| Insight Name | Error Message |
|:-:|:-:|
| ensemble-methods | Insight has a practice question but it's missing the `practiceQuestion` metadata field. |

> If you notice anything wrong with these rules contact content-team@enki.com.

**Comment by enki-hq** [comment]
### Course

| Course Name | Error Message |
|:-:|:-:|
| machine-learning | We couldn't find these workouts: `learning-by-doing`, `learning-by-chasing-a-reward`. |
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact content-team@enki.com.

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact content-team@enki.com.

**Comment by enki-hq** [comment]
### Insight previews:
- [ ] [enkidevs/curriculum/ai/machine-learning/in-practice/overfitting.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/ai/machine-learning/in-practice/overfitting.md?ref=9e7a301ed7be90810a051a94dec0573581037b0c)
- [ ] [enkidevs/curriculum/ai/machine-learning/in-practice/recap-of-ml-technologies.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/ai/machine-learning/in-practice/recap-of-ml-technologies.md?ref=9e7a301ed7be90810a051a94dec0573581037b0c)
- [ ] [enkidevs/curriculum/ai/machine-learning/in-practice/selecting-data-features.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/ai/machine-learning/in-practice/selecting-data-features.md?ref=9e7a301ed7be90810a051a94dec0573581037b0c)
- [ ] [enkidevs/curriculum/ai/machine-learning/in-practice/semi-supervised-learning.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/ai/machine-learning/in-practice/semi-supervised-learning.md?ref=9e7a301ed7be90810a051a94dec0573581037b0c)
- [ ] [enkidevs/curriculum/ai/machine-learning/learn-by-chasing-a-reward/reinforcement-learning-benefits.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/ai/machine-learning/learn-by-chasing-a-reward/reinforcement-learning-benefits.md?ref=9e7a301ed7be90810a051a94dec0573581037b0c)
- [ ] [enkidevs/curriculum/ai/machine-learning/learn-by-chasing-a-reward/reinforcement-learning-challenges.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/ai/machine-learning/learn-by-chasing-a-reward/reinforcement-learning-challenges.md?ref=9e7a301ed7be90810a051a94dec0573581037b0c)
- [ ] [enkidevs/curriculum/ai/machine-learning/learn-by-chasing-a-reward/reinforcement-learning-example.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/ai/machine-learning/learn-by-chasing-a-reward/reinforcement-learning-example.md?ref=9e7a301ed7be90810a051a94dec0573581037b0c)
- [ ] [enkidevs/curriculum/ai/machine-learning/learn-by-chasing-a-reward/what-is-reinforcement-learning.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/ai/machine-learning/learn-by-chasing-a-reward/what-is-reinforcement-learning.md?ref=9e7a301ed7be90810a051a94dec0573581037b0c)
- [ ] [enkidevs/curriculum/ai/machine-learning/learn-by-doing/unsupervised-learning-benefits.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/ai/machine-learning/learn-by-doing/unsupervised-learning-benefits.md?ref=9e7a301ed7be90810a051a94dec0573581037b0c)
- [ ] [enkidevs/curriculum/ai/machine-learning/learn-by-doing/unsupervised-learning-challenges.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/ai/machine-learning/learn-by-doing/unsupervised-learning-challenges.md?ref=9e7a301ed7be90810a051a94dec0573581037b0c)
- [ ] [enkidevs/curriculum/ai/machine-learning/learn-by-doing/unsupervised-learning-example.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/ai/machine-learning/learn-by-doing/unsupervised-learning-example.md?ref=9e7a301ed7be90810a051a94dec0573581037b0c)
- [ ] [enkidevs/curriculum/ai/machine-learning/learn-by-doing/what-is-unsupervised-learning.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/ai/machine-learning/learn-by-doing/what-is-unsupervised-learning.md?ref=9e7a301ed7be90810a051a94dec0573581037b0c)
- [ ] [enkidevs/curriculum/ai/machine-learning/learning-through-examples/supervised-learning-benefits.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/ai/machine-learning/learning-through-examples/supervised-learning-benefits.md?ref=9e7a301ed7be90810a051a94dec0573581037b0c)
- [ ] [enkidevs/curriculum/ai/machine-learning/learning-through-examples/supervised-learning-challenges.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/ai/machine-learning/learning-through-examples/supervised-learning-challenges.md?ref=9e7a301ed7be90810a051a94dec0573581037b0c)
- [ ] [enkidevs/curriculum/ai/machine-learning/learning-through-examples/supervised-learning-example.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/ai/machine-learning/learning-through-examples/supervised-learning-example.md?ref=9e7a301ed7be90810a051a94dec0573581037b0c)
- [ ] [enkidevs/curriculum/ai/machine-learning/learning-through-examples/what-is-supervised-learning.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/ai/machine-learning/learning-through-examples/what-is-supervised-learning.md?ref=9e7a301ed7be90810a051a94dec0573581037b0c)
- [ ] [enkidevs/curriculum/ai/machine-learning/models-and-methods/decision-trees.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/ai/machine-learning/models-and-methods/decision-trees.md?ref=9e7a301ed7be90810a051a94dec0573581037b0c)
- [ ] [enkidevs/curriculum/ai/machine-learning/models-and-methods/ensemble-methods.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/ai/machine-learning/models-and-methods/ensemble-methods.md?ref=9e7a301ed7be90810a051a94dec0573581037b0c)
- [ ] [enkidevs/curriculum/ai/machine-learning/models-and-methods/linear-models.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/ai/machine-learning/models-and-methods/linear-models.md?ref=9e7a301ed7be90810a051a94dec0573581037b0c)
- [ ] [enkidevs/curriculum/ai/machine-learning/neural-networks/neural-network-example.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/ai/machine-learning/neural-networks/neural-network-example.md?ref=9e7a301ed7be90810a051a94dec0573581037b0c)
- [ ] [enkidevs/curriculum/ai/machine-learning/neural-networks/neural-networks.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/ai/machine-learning/neural-networks/neural-networks.md?ref=9e7a301ed7be90810a051a94dec0573581037b0c)
- [ ] [enkidevs/curriculum/ai/machine-learning/neural-networks/simple-layers-make-complex-decisions.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/ai/machine-learning/neural-networks/simple-layers-make-complex-decisions.md?ref=9e7a301ed7be90810a051a94dec0573581037b0c)
- [ ] [enkidevs/curriculum/ai/machine-learning/neural-networks/weight-vs-bias.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/ai/machine-learning/neural-networks/weight-vs-bias.md?ref=9e7a301ed7be90810a051a94dec0573581037b0c)
- [ ] [enkidevs/curriculum/ai/machine-learning/welcome/demystifying-the-magic.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/ai/machine-learning/welcome/demystifying-the-magic.md?ref=9e7a301ed7be90810a051a94dec0573581037b0c)
- [ ] [enkidevs/curriculum/ai/machine-learning/welcome/is-ml-magic.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/ai/machine-learning/welcome/is-ml-magic.md?ref=9e7a301ed7be90810a051a94dec0573581037b0c)
- [ ] [enkidevs/curriculum/ai/machine-learning/welcome/ml-is-everywhere.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/ai/machine-learning/welcome/ml-is-everywhere.md?ref=9e7a301ed7be90810a051a94dec0573581037b0c)
- [ ] [enkidevs/curriculum/ai/machine-learning/welcome/ml-vs-ai.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/ai/machine-learning/welcome/ml-vs-ai.md?ref=9e7a301ed7be90810a051a94dec0573581037b0c)
- [ ] [enkidevs/curriculum/ai/machine-learning/welcome/why-is-ml-important.md](https://app.enki.com/public/preview-insight/enkidevs/curriculum/ai/machine-learning/welcome/why-is-ml-important.md?ref=9e7a301ed7be90810a051a94dec0573581037b0c)

**Comment by Stefan-Stojanovic** [comment]
Closing and deleting this as a newer version was published

**Pull request closed without merging**